comfy_cli/tracking.py

```
@@ -161,3 +248,14 @@ def init_tracking(enable_tracking: bool):
         logging.debug("Tracking install event.")
         config_manager.set(constants.CONFIG_KEY_INSTALL_EVENT_TRIGGERED, "True")
         track_event("install")
+
+
+def _flush_all_providers() -> None:
+    for provider in PROVIDERS:
+        try:
+            provider.flush()
+        except Exception as e:  # noqa: BLE001
```

Pylint (4.0.5)

[warning] Catching too general exception Exception

(W0718)